lib/remi/data_subjects/gsheet_file.rb

```diff
@@ -0,0 +1,163 @@
+
+module Remi
+
+  require 'google/apis/sheets_v4'
+  require 'google/apis/drive_v3'
+  require 'googleauth'
+  require 'googleauth/stores/file_token_store'
+  require 'daru'
+  require 'fileutils'
```

Comment:
I'd prefer `fileutils` to be required under the "core libraries" section of `lib/remi.rb`